windows/src/buildtools/mkver/Main.pas

```diff
@@ -175,95 +215,59 @@ procedure UpdateResource;
 
   // I'm free!!!  I'm free!!!  I'm finally free!!  Thank God almighty I'm free at last! (end).
```

Comment:
... need a little help, there?

Reply:
😁 That might have been another family member "helping" many years ago!